composer.json

```diff
@@ -33,7 +33,7 @@
         "doctrine/doctrine-fixtures-bundle": "~2.2",
         "doctrine/doctrine-migrations-bundle": "~1.0",
         "doctrine/orm": ">=2.4.8,<2.5",
-        "doctrine/phpcr-odm": "^1.2.2",
+        "doctrine/phpcr-odm": "^1.3@rc",
```

Comment:
For PHP7 support - `<1.3.0` contains documents annotated with `@String` which we don't really use. `cache:warmup` generates proxies for all Doctrine entities and documents if `doctrine.orm.auto_generate_proxy_classes` is `false`. Sylius sets it to `%kernel.debug%`.

When `kernel.debug` is:
 - `true`: proxies are lazy-generated, only when needed (`@String` annotation is never parsed)
 - `false`: proxies are generated only once at warmup